ui/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@radix-ui/react-checkbox": "^1.0.3",
-    "@radix-ui/react-dialog": "^1.0.3",
+    "@radix-ui/react-dialog": "^1.0.4-rc.9",
```

Comment:
Early update because I wanted https://github.com/radix-ui/primitives/pull/2145 for the multistep form